src/_pytest/main.py

```diff
@@ -78,6 +78,11 @@ def pytest_addoption(parser: Parser) -> None:
         type="args",
         default=[],
     )
+    parser.addini(
+        "discover_imports",
```

Comment:
Is the idea that `discover_imports` is really only active if 'testpaths' is also defined in `pytest.ini`?